src/assets/js/protocol/protocol-navigation.js

```diff
@@ -47,18 +49,23 @@ if (typeof Mzp === 'undefined') { // eslint-disable-line block-scoped-var
      * Set initial ARIA navigation states.
      */
     Navigation.setAria = function() {
-        navItems.setAttribute('aria-expanded', false);
+        for (var i = 0; i < navItemsLists.length; i++) {
+            navItemsLists[i].setAttribute('aria-expanded', false);
+        }
     };
 
     /**
      * Bind navigation event handlers.
      */
     Navigation.bindEvents = function() {
-        navItems = document.querySelector('.mzp-c-navigation-items');
-        if (navItems) {
-            document.querySelector('.mzp-c-navigation-menu-button').addEventListener('click', Navigation.onClick, false);
-            Navigation.setAria();
+        navItemsLists = document.querySelectorAll('.mzp-c-navigation-items');
+        if (navItemsLists.length > 0) {
+            var navButtons = document.querySelectorAll('.mzp-c-navigation-menu-button');
+            for (var i = 0; i < navButtons.length; i++) {
+                navButtons[i].addEventListener('click', Navigation.onClick, false);
+            }
         }
+        Navigation.setAria();
```

Comment:
Could `setAria()` be inside the `if (navItemsLists.length > 0) {}` conditional?

Reply:
✅